flow.js

```diff
@@ -11,6 +11,11 @@ type aliaser<Token> = {
   alias: (sourceToken: Token, destToken: Token) => aliaser<*>,
 };
 
+declare type Token<T> = {
```

Comment:
+1 good catch!  Not a huge fan of `flow.js` and Flow's lack of overrides otherwise, but agreed that we should be consistent in this package.